modules/runner/src/integrationTest/java/org/apache/ignite/internal/sql/engine/ItDmlTest.java

```diff
@@ -107,13 +106,7 @@ public void mergeOpChangePrimaryKey() {
                 + "WHEN MATCHED THEN UPDATE SET b = src.b, k1 = src.k1 "
                 + "WHEN NOT MATCHED THEN INSERT (k1, k2, a, b) VALUES (src.k1, src.k2, src.a, src.b)";
 
-        sql(sql);
-
-        assertQuery("SELECT * FROM test2 ORDER BY k1")
-                .returns(222, 222, 1, 300, null)
-                .returns(111, 333, 0, 100, "")
-                .returns(444, 444, 2, 200, null)
-                .check();
+        assertThrows(CalciteContextException.class, () -> sql(sql));
```

Comment:
Let's add the check SQL code here.

Reply:
We have to throw SqlException to check error code, but CalciteContextException is thrown.